### DISCUSSION TITLE
Spanish language appears as 100% translated while still has untranslated strings

### DISCUSSION THREAD

**Discussion by greenled** [encode/django-rest-framework#7879] [Potential Issue]
I found there are some untranslated strings in the Spanish [.po](https://github.com/encode/django-rest-framework/blob/master/rest_framework/locale/es/LC_MESSAGES/django.po) file. Tried to use Transifex to translate them, as recommended by the docs, but Spanish language for [django-rest-framework project](https://www.transifex.com/django-rest-framework-1/django-rest-framework/) shows as 100% translated (0 strings to translate).

Is Transifex integration working?

Should I make a PR with the translated strings?

**Comment by lovelydinosaur** [comment]
That probably means we need to upload the latest .po to transifex.

And nope, best not to do so via PR, but thanks.

**Reply by greenled** [reply]
You're welcome
